### PR TITLE
Add support for f-strings

### PR DESCRIPTION
**Pull request by kirbyfan64**
It's a bit ugly, but hey, it works, right? ;)

### CODE REVIEW

**Comment by kirbyfan64** [comment]
FWIW, it'd also be nice to have a new release soon (#53), since I'm working on f-string support for Hy, and it's going to need this...

File: tests/test_code_gen.py [hunk 1]
```diff
@@ -353,5 +353,13 @@ def test_elif(self):
         """
         self.assertAstSourceEqual(source)
 
+    def test_fstrings(self):
+        source = """
+        f'{x}'
```
Comment (berkerpeksag) [inline tests/test_code_gen.py]: Could you also add tests for `f'{x.y}'` and `f'{int(x}'`?

**Comment by kirbyfan64** [comment]
@berkerpeksag Done.

**Comment by berkerpeksag** [comment]
Sorry for the late merge, thanks!

**Pull request merged**